### ISSUE TITLE
The UI stops updating for large games. 

### ISSUE THREAD

**Issue by Domkeykong** [Heroic-Games-Launcher/HeroicGamesLauncher#1705]
### Describe the bug

The UI stops updating for large games. 

### Add logs

```
Heroic Version: 2.3.10 Brook
Legendary Version:  0.20.27 Dark Energy (hotfix)
OS: Freedesktop.org KERNEL: 5.13.0-valve21-1-neptune-02209-g2a5bdc1102a0 ARCH: x64
CPU: AMD Custom APU 0405 @2.11 GOVERNOR: schedutil
RAM: Total: 14.49 GiB Available: 11.36 GiB
GRAPHICS: GPU0: Device 163f VRAM: 512MB DRIVER:  
PROTOCOL: x11

(17:42:09) INFO:    [Gog]:             Getting data about the user
(17:42:09) WARNING: [Backend]:         Failed to register protocol with OS.
(17:42:09) INFO:    [Gog]:             Saved user data to config
(17:42:10) INFO:    [Frontend]:        Refreshing Library
(17:42:10) INFO:    [Legendary]:       Refreshing library...
(17:42:10) INFO:    [Legendary]:       Refreshing Epic Games...
(17:42:10) INFO:    [Gog]:             Getting GOG library
(17:42:11) INFO:    [Gog]:             Number of library pages: 1
(17:42:11) INFO:    [Gog]:             Saved games data
(17:42:11) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list
(17:42:13) INFO:    [Legendary]:       Updating game list
(17:42:13) INFO:    [Legendary]:       Game List Updated
(17:42:13) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:42:13) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:42:13) INFO:    [Backend]:         Downloaded Winetricks
(17:42:13) INFO:    [Legendary]:       Checking for game updates: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(17:42:13) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(17:42:15) INFO:    [Legendary]:       Found 0 game(s) to update
(17:42:15) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:42:15) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:42:15) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:42:15) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:42:16) INFO:    [Gog]:             Found 0 game(s) to update
(17:43:03) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:03) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:03) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:03) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:06) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:06) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:21) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:21) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(17:43:21) INFO:    [Gog]:             Installing 1439487606 with: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl download 1439487606 --platform linux --path=/home/deck/Games/GOG --token <redacted> --skip-dlcs --lang=en
(17:43:21) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl download 1439487606 --platform linux --path=/home/deck/Games/GOG --token <redacted> --skip-dlcs --lang=en
(17:43:21) DEBUG:   [Gog]:             Logging to file "/home/deck/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/1439487606.log"
(17:43:41) INFO:    [Backend]:         Progress for 1439487606: 0.1%/11.70MiB/06:56:05
(17:43:42) INFO:    [Backend]:         Progress for 1439487606: 0.1%/11.70MiB/06:56:06
(17:43:43) INFO:    [Backend]:         Progress for 1439487606: 0.1%/11.70MiB/06:56:07
(17:43:44) INFO:    [Backend]:         Progress for 1439487606: 0.1%/11.70MiB/06:56:08
(17:43:45) INFO:    [Backend]:         Progress for 1439487606: 0.1%/11.70MiB/06:56:09
(17:43:48) INFO:    [Backend]:         Progress for 1439487606: 0.2%/23.40MiB/16:39:12
(17:43:49) INFO:    [Backend]:         Progress for 1439487606: 0.2%/23.40MiB/16:39:13
(17:43:50) INFO:    [Backend]:         Progress for 1439487606: 0.2%/23.40MiB/16:39:14
(17:43:51) INFO:    [Backend]:         Progress for 1439487606: 0.2%/23.40MiB/16:39:15
(17:43:52) INFO:    [Backend]:         Progress for 1439487606: 0.2%/23.40MiB/16:39:16
(17:43:56) INFO:    [Backend]:         Progress for 1439487606: 0.3%/35.11MiB/24:57:18
(17:43:56) INFO:    [Backend]:         Progress for 1439487606: 0.3%/35.11MiB/24:57:19
(17:43:56) INFO:    [Backend]:         Progress for 1439487606: 0.3%/35.11MiB/24:57:20
(17:43:57) INFO:    [Backend]:         Progress for 1439487606: 0.4%/46.81MiB/13:25:21
(17:43:58) INFO:    [Backend]:         Progress for 1439487606: 0.4%/46.81MiB/13:25:22
(17:44:03) INFO:    [Backend]:         Progress for 1439487606: 0.4%/46.81MiB/29:05:23
(17:44:03) INFO:    [Backend]:         Progress for 1439487606: 0.4%/46.81MiB/29:05:24
(17:44:04) INFO:    [Backend]:         Progress for 1439487606: 0.5%/58.51MiB/38:45:28
(17:44:05) INFO:    [Backend]:         Progress for 1439487606: 0.5%/58.51MiB/38:45:29
(17:44:06) INFO:    [Backend]:         Progress for 1439487606: 0.6%/70.21MiB/20:01:30
(17:44:07) INFO:    [Backend]:         Progress for 1439487606: 0.6%/70.21MiB/20:01:31
(17:44:08) INFO:    [Backend]:         Progress for 1439487606: 0.6%/70.21MiB/20:01:32
(17:44:09) INFO:    [Backend]:         Progress for 1439487606: 0.7%/81.91MiB/21:42:33
(17:44:10) INFO:    [Backend]:         Progress for 1439487606: 0.7%/81.91MiB/21:42:34
(17:44:12) INFO:    [Backend]:         Progress for 1439487606: 0.7%/81.91MiB/45:34:36
(17:44:13) INFO:    [Backend]:         Progress for 1439487606: 0.8%/93.61MiB/24:04:37
(17:44:14) INFO:    [Backend]:         Progress for 1439487606: 0.8%/93.61MiB/51:03:38
(17:44:15) INFO:    [Backend]:         Progress for 1439487606: 0.8%/93.61MiB/51:03:39
(17:44:16) INFO:    [Backend]:         Progress for 1439487606: 0.9%/105.32MiB/26:29:40
(17:44:17) INFO:    [Backend]:         Progress for 1439487606: 0.9%/105.32MiB/26:29:41
(17:44:18) INFO:    [Backend]:         Progress for 1439487606: 0.9%/105.32MiB/55:08:42
(17:44:19) INFO:    [Backend]:         Progress for 1439487606: 0.9%/105.32MiB/55:08:43
(17:44:20) INFO:    [Backend]:         Progress for 1439487606: 1%/117.02MiB/28:51:44
(17:44:21) INFO:    [Backend]:         Progress for 1439487606: 1%/117.02MiB/60:35:45
(17:44:22) INFO:    [Backend]:         Progress for 1439487606: 1%/117.02MiB/60:35:46
(17:44:23) INFO:    [Backend]:         Progress for 1439487606: 1%/117.02MiB/60:35:47
(17:44:24) INFO:    [Backend]:         Progress for 1439487606: 1%/117.02MiB/60:35:48
(17:44:25) INFO:    [Backend]:         Progress for 1439487606: 1.1%/128.72MiB/67:24:49
(17:44:26) INFO:    [Backend]:         Progress for 1439487606: 1.1%/128.72MiB/67:24:50
(17:44:27) INFO:    [Backend]:         Progress for 1439487606: 1.1%/128.72MiB/67:24:51
(17:44:28) INFO:    [Backend]:         Progress for 1439487606: 1.1%/128.72MiB/67:24:52
(17:44:29) INFO:    [Backend]:         Progress for 1439487606: 1.2%/140.42MiB/34:59:53
(17:44:30) INFO:    [Backend]:         Progress for 1439487606: 1.2%/140.42MiB/72:49:54
(17:44:31) INFO:    [Backend]:         Progress for 1439487606: 1.2%/140.42MiB/72:49:55
(17:44:32) INFO:    [Backend]:         Progress for 1439487606: 1.2%/140.42MiB/72:49:56
(17:44:33) INFO:    [Backend]:         Progress for 1439487606: 1.3%/152.12MiB/37:42:57
(17:44:34) INFO:    [Backend]:         Progress for 1439487606: 1.3%/152.12MiB/78:14:58
(17:44:35) INFO:    [Backend]:         Progress for 1439487606: 1.3%/152.12MiB/78:14:59
(17:44:36) INFO:    [Backend]:         Progress for 1439487606: 1.4%/163.83MiB/40:05:00
(17:44:37) INFO:    [Backend]:         Progress for 1439487606: 1.4%/163.83MiB/40:05:01
(17:44:38) INFO:    [Backend]:         Progress for 1439487606: 1.4%/163.83MiB/82:17:02
(17:44:39) INFO:    [Backend]:         Progress for 1439487606: 1.4%/163.83MiB/82:17:03
(17:44:40) INFO:    [Backend]:         Progress for 1439487606: 1.5%/175.53MiB/42:25:04
(17:44:41) INFO:    [Backend]:         Progress for 1439487606: 1.5%/175.53MiB/87:40:05
(17:44:42) INFO:    [Backend]:         Progress for 1439487606: 1.5%/175.53MiB/87:40:06
(17:44:43) INFO:    [Backend]:         Progress for 1439487606: 1.6%/187.23MiB/44:47:07
(17:44:44) INFO:    [Backend]:         Progress for 1439487606: 1.6%/187.23MiB/44:47:08
(17:44:45) INFO:    [Backend]:         Progress for 1439487606: 1.6%/187.23MiB/91:41:09
(17:44:46) INFO:    [Backend]:         Progress for 1439487606: 1.7%/198.93MiB/46:48:10
(17:44:47) INFO:    [Backend]:         Progress for 1439487606: 1.7%/198.93MiB/46:48:11
(17:44:48) INFO:    [Backend]:         Progress for 1439487606: 1.7%/198.93MiB/95:42:12
(17:44:49) INFO:    [Backend]:         Progress for 1439487606: 1.8%/210.63MiB/48:48:13
(17:44:51) INFO:    [Backend]:         Progress for 1439487606: 1.8%/210.63MiB/99:42:15
(17:44:52) INFO:    [Backend]:         Progress for 1439487606: 1.9%/222.33MiB/50:48:16
(17:44:53) INFO:    [Backend]:         Progress for 1439487606: 1.9%/222.33MiB/50:48:17
(17:44:55) INFO:    [Backend]:         Progress for 1439487606: 2%/234.04MiB/52:47:19
(17:44:56) INFO:    [Backend]:         Progress for 1439487606: 2%/234.04MiB/52:47:20
(17:44:58) INFO:    [Backend]:         Progress for 1439487606: 2.1%/245.74MiB/54:47:22
(17:44:59) INFO:    [Backend]:         Progress for 1439487606: 2.1%/245.74MiB/54:47:23
(17:45:01) INFO:    [Backend]:         Progress for 1439487606: 2.2%/257.44MiB/56:46:25
(17:45:02) INFO:    [Backend]:         Progress for 1439487606: 2.2%/257.44MiB/56:46:26
(17:45:04) INFO:    [Backend]:         Progress for 1439487606: 2.3%/269.14MiB/58:45:28
(17:45:05) INFO:    [Backend]:         Progress for 1439487606: 2.3%/269.14MiB/58:45:29
(17:45:07) INFO:    [Backend]:         Progress for 1439487606: 2.4%/280.84MiB/60:43:31
(17:45:08) INFO:    [Backend]:         Progress for 1439487606: 2.4%/280.84MiB/60:43:32
(17:45:10) INFO:    [Backend]:         Progress for 1439487606: 2.5%/292.54MiB/62:41:34
(17:45:11) INFO:    [Backend]:         Progress for 1439487606: 2.5%/292.54MiB/62:41:35
(17:45:13) INFO:    [Backend]:         Progress for 1439487606: 2.6%/304.25MiB/64:40:37
(17:45:14) INFO:    [Backend]:         Progress for 1439487606: 2.6%/304.25MiB/64:40:38
(17:45:16) INFO:    [Backend]:         Progress for 1439487606: 2.7%/315.95MiB/66:37:40
(17:45:17) INFO:    [Backend]:         Progress for 1439487606: 2.7%/315.95MiB/66:37:41
(17:45:19) INFO:    [Backend]:         Progress for 1439487606: 2.8%/327.65MiB/68:35:43
(17:45:20) INFO:    [Backend]:         Progress for 1439487606: 2.8%/327.65MiB/68:35:44
(17:45:21) INFO:    [Backend]:         Progress for 1439487606: 2.9%/339.35MiB/70:13:45
(17:45:22) INFO:    [Backend]:         Progress for 1439487606: 2.9%/339.35MiB/70:13:46
(17:45:23) INFO:    [Backend]:         Progress for 1439487606: 2.9%/339.35MiB/70:13:47
(17:45:25) INFO:    [Backend]:         Progress for 1439487606: 3%/351.05MiB/71:49:49
(17:45:27) INFO:    [Backend]:         Progress for 1439487606: 3.1%/362.76MiB/73:46:51
(17:45:28) INFO:    [Backend]:         Progress for 1439487606: 3.1%/362.76MiB/73:46:52
(17:45:30) INFO:    [Backend]:         Progress for 1439487606: 3.2%/374.46MiB/75:42:54
(17:45:31) INFO:    [Backend]:         Progress for 1439487606: 3.2%/374.46MiB/75:42:55
(17:45:33) INFO:    [Backend]:         Progress for 1439487606: 3.3%/386.16MiB/77:39:57
(17:45:34) INFO:    [Backend]:         Progress for 1439487606: 3.3%/386.16MiB/77:39:58
(17:45:36) INFO:    [Backend]:         Progress for 1439487606: 3.4%/397.86MiB/79:35:00
(17:45:37) INFO:    [Backend]:         Progress for 1439487606: 3.4%/397.86MiB/79:35:01
(17:45:40) INFO:    [Backend]:         Progress for 1439487606: 3.5%/409.56MiB/81:50:04
(17:45:44) INFO:    [Backend]:         Progress for 1439487606: 3.6%/421.26MiB/84:26:08
(17:45:47) INFO:    [Backend]:         Progress for 1439487606: 3.7%/432.97MiB/86:42:11
(17:45:48) INFO:    [Backend]:         Progress for 1439487606: 3.7%/432.97MiB/86:42:12
(17:45:50) INFO:    [Backend]:         Progress for 1439487606: 3.8%/444.67MiB/88:37:14
(17:45:51) INFO:    [Backend]:         Progress for 1439487606: 3.8%/444.67MiB/88:37:15
(17:45:54) INFO:    [Backend]:         Progress for 1439487606: 3.9%/456.37MiB/90:51:18
(17:45:57) INFO:    [Backend]:         Progress for 1439487606: 4%/468.07MiB/93:06:21
(17:45:58) INFO:    [Backend]:         Progress for 1439487606: 4%/468.07MiB/93:06:22
(17:46:00) INFO:    [Backend]:         Progress for 1439487606: 4.1%/479.77MiB/95:01:24
(17:46:01) INFO:    [Backend]:         Progress for 1439487606: 4.1%/479.77MiB/95:01:25
(17:46:03) INFO:    [Backend]:         Progress for 1439487606: 4.2%/491.48MiB/96:55:27
(17:46:04) INFO:    [Backend]:         Progress for 1439487606: 4.2%/491.48MiB/96:55:28
(17:46:06) INFO:    [Backend]:         Progress for 1439487606: 4.3%/503.18MiB/98:48:30
(17:46:07) INFO:    [Backend]:         Progress for 1439487606: 4.3%/503.18MiB/98:48:31
(18:32:41) INFO:    [Backend]:         Progress for 1439487606: 95.2%/11140.11MiB/98:07:05
(18:32:42) INFO:    [Backend]:         Progress for 1439487606: 95.2%/11140.11MiB/98:07:06
(18:32:43) INFO:    [Backend]:         Progress for 1439487606: 95.3%/11151.81MiB/96:09:07
(18:32:44) INFO:    [Backend]:         Progress for 1439487606: 95.3%/11151.81MiB/96:09:08
(18:32:45) INFO:    [Backend]:         Progress for 1439487606: 95.3%/11151.81MiB/96:09:09
(18:32:46) INFO:    [Backend]:         Progress for 1439487606: 95.4%/11163.51MiB/94:11:10
(18:32:47) INFO:    [Backend]:         Progress for 1439487606: 95.4%/11163.51MiB/94:11:11
(18:32:49) INFO:    [Backend]:         Progress for 1439487606: 95.5%/11175.22MiB/92:14:13
(18:32:50) INFO:    [Backend]:         Progress for 1439487606: 95.5%/11175.22MiB/92:14:14
(18:32:52) INFO:    [Backend]:         Progress for 1439487606: 95.6%/11186.92MiB/90:16:16
(18:32:53) INFO:    [Backend]:         Progress for 1439487606: 95.6%/11186.92MiB/90:16:17
(18:32:55) INFO:    [Backend]:         Progress for 1439487606: 95.7%/11198.62MiB/88:19:19
(18:32:56) INFO:    [Backend]:         Progress for 1439487606: 95.7%/11198.62MiB/88:19:20
(18:32:58) INFO:    [Backend]:         Progress for 1439487606: 95.8%/11210.32MiB/86:21:22
(18:32:59) INFO:    [Backend]:         Progress for 1439487606: 95.8%/11210.32MiB/86:21:23
(18:33:01) INFO:    [Backend]:         Progress for 1439487606: 95.9%/11222.02MiB/84:22:25
(18:33:02) INFO:    [Backend]:         Progress for 1439487606: 95.9%/11222.02MiB/84:22:26
(18:33:03) INFO:    [Backend]:         Progress for 1439487606: 96%/11233.72MiB/82:23:27
(18:33:04) INFO:    [Backend]:         Progress for 1439487606: 96%/11233.72MiB/82:23:28
(18:33:05) INFO:    [Backend]:         Progress for 1439487606: 96%/11233.72MiB/82:23:29
(18:33:07) INFO:    [Backend]:         Progress for 1439487606: 96.1%/11245.43MiB/80:24:31
(18:33:10) INFO:    [Backend]:         Progress for 1439487606: 96.2%/11257.13MiB/78:26:34
(18:33:11) INFO:    [Backend]:         Progress for 1439487606: 96.2%/11257.13MiB/78:26:35
(18:33:13) INFO:    [Backend]:         Progress for 1439487606: 96.3%/11268.83MiB/76:27:37
(18:33:16) INFO:    [Backend]:         Progress for 1439487606: 96.4%/11280.53MiB/74:27:40
(18:33:17) INFO:    [Backend]:         Progress for 1439487606: 96.4%/11280.53MiB/74:27:41
(18:33:20) INFO:    [Backend]:         Progress for 1439487606: 96.5%/11292.23MiB/72:28:44
(18:33:22) INFO:    [Backend]:         Progress for 1439487606: 96.6%/11303.94MiB/70:28:46
(18:33:27) INFO:    [Backend]:         Progress for 1439487606: 96.6%/11303.94MiB/70:28:47
(18:33:31) INFO:    [Backend]:         Progress for 1439487606: 96.8%/11327.34MiB/66:31:55
(18:33:32) INFO:    [Backend]:         Progress for 1439487606: 96.8%/11327.34MiB/66:31:56
(18:33:34) INFO:    [Backend]:         Progress for 1439487606: 96.9%/11339.04MiB/64:30:58
(18:33:35) INFO:    [Backend]:         Progress for 1439487606: 96.9%/11339.04MiB/64:30:59
(18:33:37) INFO:    [Backend]:         Progress for 1439487606: 97%/11350.74MiB/62:29:01
(18:33:41) INFO:    [Backend]:         Progress for 1439487606: 97.1%/11362.44MiB/60:28:05
(18:33:44) INFO:    [Backend]:         Progress for 1439487606: 97.2%/11374.15MiB/58:27:08
(18:33:45) INFO:    [Backend]:         Progress for 1439487606: 97.2%/11374.15MiB/58:27:09
(18:33:46) INFO:    [Backend]:         Progress for 1439487606: 97.3%/11385.85MiB/56:25:10
(18:33:47) INFO:    [Backend]:         Progress for 1439487606: 97.3%/11385.85MiB/56:25:11
(18:33:48) INFO:    [Backend]:         Progress for 1439487606: 97.3%/11385.85MiB/56:25:12
(18:33:50) INFO:    [Backend]:         Progress for 1439487606: 97.4%/11397.55MiB/54:23:14
(18:33:52) INFO:    [Backend]:         Progress for 1439487606: 97.5%/11409.25MiB/52:20:16
(18:33:53) INFO:    [Backend]:         Progress for 1439487606: 97.5%/11409.25MiB/52:20:17
(18:33:54) INFO:    [Backend]:         Progress for 1439487606: 97.5%/11409.25MiB/52:20:18
(18:33:55) INFO:    [Backend]:         Progress for 1439487606: 97.6%/11420.95MiB/50:17:19
(18:33:56) INFO:    [Backend]:         Progress for 1439487606: 97.6%/11420.95MiB/50:17:20
(18:33:58) INFO:    [Backend]:         Progress for 1439487606: 97.7%/11432.66MiB/48:14:22
(18:33:59) INFO:    [Backend]:         Progress for 1439487606: 97.7%/11432.66MiB/48:14:23
(18:34:00) INFO:    [Backend]:         Progress for 1439487606: 97.7%/11432.66MiB/96:32:24
(18:34:01) INFO:    [Backend]:         Progress for 1439487606: 97.8%/11444.36MiB/46:11:25
(18:34:02) INFO:    [Backend]:         Progress for 1439487606: 97.8%/11444.36MiB/46:11:26
(18:34:04) INFO:    [Backend]:         Progress for 1439487606: 97.9%/11456.06MiB/44:08:28
(18:34:05) INFO:    [Backend]:         Progress for 1439487606: 97.9%/11456.06MiB/44:08:29
(18:34:06) INFO:    [Backend]:         Progress for 1439487606: 97.9%/11456.06MiB/88:19:30
(18:34:07) INFO:    [Backend]:         Progress for 1439487606: 98%/11467.76MiB/42:04:31
(18:34:08) INFO:    [Backend]:         Progress for 1439487606: 98%/11467.76MiB/42:04:32
(18:34:09) INFO:    [Backend]:         Progress for 1439487606: 98%/11467.76MiB/84:12:33
(18:34:10) INFO:    [Backend]:         Progress for 1439487606: 98.1%/11479.46MiB/40:00:34
(18:34:11) INFO:    [Backend]:         Progress for 1439487606: 98.1%/11479.46MiB/40:00:35
(18:34:12) INFO:    [Backend]:         Progress for 1439487606: 98.2%/11491.16MiB/37:56:36
(18:34:13) INFO:    [Backend]:         Progress for 1439487606: 98.2%/11491.16MiB/37:56:37
(18:34:14) INFO:    [Backend]:         Progress for 1439487606: 98.2%/11491.16MiB/37:56:38
(18:34:15) INFO:    [Backend]:         Progress for 1439487606: 98.3%/11502.87MiB/35:51:39
(18:34:16) INFO:    [Backend]:         Progress for 1439487606: 98.3%/11502.87MiB/35:51:40
(18:34:17) INFO:    [Backend]:         Progress for 1439487606: 98.3%/11502.87MiB/71:45:41
(18:34:18) INFO:    [Backend]:         Progress for 1439487606: 98.4%/11514.57MiB/33:47:42
(18:34:19) INFO:    [Backend]:         Progress for 1439487606: 98.4%/11514.57MiB/33:47:43
(18:34:20) INFO:    [Backend]:         Progress for 1439487606: 98.4%/11514.57MiB/67:36:44
(18:34:21) INFO:    [Backend]:         Progress for 1439487606: 98.5%/11526.27MiB/31:42:45
(18:34:22) INFO:    [Backend]:         Progress for 1439487606: 98.5%/11526.27MiB/31:42:46
(18:34:23) INFO:    [Backend]:         Progress for 1439487606: 98.5%/11526.27MiB/63:26:47
(18:34:24) INFO:    [Backend]:         Progress for 1439487606: 98.6%/11537.97MiB/29:37:48
(18:34:25) INFO:    [Backend]:         Progress for 1439487606: 98.6%/11537.97MiB/29:37:49
(18:34:26) INFO:    [Backend]:         Progress for 1439487606: 98.6%/11537.97MiB/59:16:50
(18:34:27) INFO:    [Backend]:         Progress for 1439487606: 98.7%/11549.67MiB/27:31:51
(18:34:28) INFO:    [Backend]:         Progress for 1439487606: 98.7%/11549.67MiB/27:31:52
(18:34:29) INFO:    [Backend]:         Progress for 1439487606: 98.7%/11549.67MiB/55:05:53
(18:34:30) INFO:    [Backend]:         Progress for 1439487606: 98.8%/11561.37MiB/25:26:54
(18:34:31) INFO:    [Backend]:         Progress for 1439487606: 98.8%/11561.37MiB/25:26:55
(18:34:32) INFO:    [Backend]:         Progress for 1439487606: 98.8%/11561.37MiB/50:54:56
(18:34:33) INFO:    [Backend]:         Progress for 1439487606: 98.9%/11573.08MiB/23:20:57
(18:34:34) INFO:    [Backend]:         Progress for 1439487606: 98.9%/11573.08MiB/23:20:58
(18:34:35) INFO:    [Backend]:         Progress for 1439487606: 98.9%/11573.08MiB/46:42:59
(18:34:36) INFO:    [Backend]:         Progress for 1439487606: 99%/11584.78MiB/21:14:00
(18:34:37) INFO:    [Backend]:         Progress for 1439487606: 99%/11584.78MiB/21:14:01
(18:34:39) INFO:    [Backend]:         Progress for 1439487606: 99.1%/11596.48MiB/19:08:03
(18:34:40) INFO:    [Backend]:         Progress for 1439487606: 99.1%/11596.48MiB/19:08:04
(18:34:41) INFO:    [Backend]:         Progress for 1439487606: 99.2%/11608.18MiB/17:01:05
(18:34:42) INFO:    [Backend]:         Progress for 1439487606: 99.2%/11608.18MiB/17:01:06
(18:34:43) INFO:    [Backend]:         Progress for 1439487606: 99.2%/11608.18MiB/17:01:07
(18:34:44) INFO:    [Backend]:         Progress for 1439487606: 99.3%/11619.88MiB/14:54:08
(18:34:45) INFO:    [Backend]:         Progress for 1439487606: 99.3%/11619.88MiB/14:54:09
(18:34:46) INFO:    [Backend]:         Progress for 1439487606: 99.4%/11631.59MiB/12:47:10
(18:34:47) INFO:    [Backend]:         Progress for 1439487606: 99.4%/11631.59MiB/12:47:11
(18:34:48) INFO:    [Backend]:         Progress for 1439487606: 99.4%/11631.59MiB/12:47:12
(18:34:49) INFO:    [Backend]:         Progress for 1439487606: 99.5%/11643.29MiB/10:39:13
(18:34:50) INFO:    [Backend]:         Progress for 1439487606: 99.5%/11643.29MiB/10:39:14
(18:34:51) INFO:    [Backend]:         Progress for 1439487606: 99.5%/11643.29MiB/21:20:15
(18:34:52) INFO:    [Backend]:         Progress for 1439487606: 99.6%/11654.99MiB/08:32:16
(18:34:56) INFO:    [Backend]:         Progress for 1439487606: 99.7%/11666.69MiB/06:24:20
(18:34:57) INFO:    [Backend]:         Progress for 1439487606: 99.8%/11678.39MiB/04:16:21
(18:34:58) INFO:    [Backend]:         Progress for 1439487606: 99.8%/11678.39MiB/04:16:22
(18:34:59) INFO:    [Backend]:         Progress for 1439487606: 99.8%/11678.39MiB/08:33:23
(18:35:00) INFO:    [Backend]:         Progress for 1439487606: 99.9%/11690.09MiB/02:08:24
(18:35:02) INFO:    [Backend]:         Progress for 1439487606: 99.9%/11690.09MiB/04:17:26
(18:35:03) INFO:    [Backend]:         Progress for 1439487606: 100%/11701.80MiB/00:00:27
(18:39:45) INFO:    [Gog]:             Getting game metadata: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(18:39:45) DEBUG:   [Gog]:             Running GOGDL command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl info 1439487606 --token <redacted> --lang=en-US --os windows
(18:39:45) INFO:    [Backend]:         operation interrupted
(18:39:49) INFO:    [Backend]:         finished installing
(18:39:49) INFO:    [Frontend]:        Refreshing Library
(18:39:49) INFO:    [Gog]:             Getting GOG library
(18:39:50) INFO:    [Gog]:             Number of library pages: 1
(18:39:50) INFO:    [Gog]:             Saved games data
```

[1439487606.log](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/9310515/1439487606.log)

### Steps to reproduce

1. Try Installing a large Game
2. Wait
3. Wait
4. Maybe give up

### Expected behavior

The UI is updated properly

### System Information

- OS SteamOS
- Heroic Version 2.3.10

**Comment by flavioislima** [comment]
Can you try on the latest. 2.4.0 release?

**Comment by Nocccer** [comment]
Know problem for gog games. I don't know why this happens

**Comment by Domkeykong** [comment]
The issue persists on the latest version!

**Comment by flavioislima** [comment]
Only on steam deck? I tried to replicate it downloading Cyberpunk and I could not.
It is weird but let's do some more deep investigation on that.

**Comment by Nocccer** [comment]
Only happens on flatpak, it seems.

**Comment by Domkeykong** [comment]
well i tried it both on flatpak and the AppImage. flatpak was on the deck and appimage on my desktop.
I was using SOMA from gog.

**Comment by Domkeykong** [comment]
it seems a bit random when it stops updating, sometimes around 5% but also at 12% or 15%.

**Comment by Jeff-The-Geek** [comment]
Still seeing htis problem. My particular situation: 

Steam Deck
2.4.0 Flatpak
Desktop  Mode
Installing GOG Horizon Zero Dawn.

**Comment by b-m-f** [comment]
For me it stop updating as soon as the LOG has another message than the download progress.

```
(17:09:35) INFO:    [Gog]:             Progress for 1296690054: 37.2%/17185.59MiB/85:06:32
(17:09:36) INFO:    [Gog]:             Progress for 1296690054: 37.2%/17185.59MiB/92:38:33
(17:09:38) INFO:    [Gog]:             Progress for 1296690054: 37.23%/17202.21MiB/94:14:35
(17:09:39) INFO:    [Gog]:             Progress for 1296690054: 37.23%/17202.21MiB/97:51:36
(17:09:42) INFO:    [Gog]:             Progress for 1296690054: 37.23%/17202.21MiB/95:35:39
(17:09:45) INFO:    [Gog]:             Progress for 1296690054: 37.31%/17239.46MiB/95:16:42
(17:10:15) INFO:    [Legendary]:       Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(17:10:15) INFO:    [Gog]:             Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --version
(17:10:34) INFO:    [Legendary]:       Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(17:10:34) INFO:    [Gog]:             Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --version

```

**Comment by b-m-f** [comment]
I have just checked https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/gog/games.ts.

So the Event to update the UI happens in the https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/gog/games.ts#L194 `onOutput` function.

Since the logfile is involved, maybe there is a parsing error happening?

I do not have the debug version on my machine. Is there any way to get the Dev console running in the normal version so I could look for errors?

**Comment by Nocccer** [comment]
> I have just checked https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/gog/games.ts.
> 
> So the Event to update the UI happens in the https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/electron/gog/games.ts#L194 `onOutput` function.
> 
> Since the logfile is involved, maybe there is a parsing error happening?
> 
> I do not have the debug version on my machine. Is there any way to get the Dev console running in the normal version so I could look for errors? 

If you mean with dev console the chromium xonaoe you can bring it up via tray icon.

**Comment by flavioislima** [comment]
fixed on 2.4.3
